common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -870,6 +870,15 @@ def test_course_home(self):
         self.course_home_page.outline.resume_course_from_outline()
         self.assertTrue(self.courseware_page.nav.is_on_section('Test Section 2', 'Test Subsection 3'))
 
+        # Test that events are emitted when outline sections are clicked
```

Comment:
Great question. In general we are doing too much in Bok Choy, so I'm in favor of not adding these extra tests. I believe that there are other Bok Choy tests that are verifying that the overall client-side eventing is working.